client/src/pages/Tree/TreeImage.js

```diff
@@ -25,7 +27,20 @@ export const ImageLoad = React.memo(({ src, placeholder, alt = '' }) => {
       }}
       placeholder={placeholder}
       alt={alt}
+      loading="lazy"
     />
   );
 });
 ImageLoad.displayName = 'ImageLoad';
+
+export const fixScientificForImage = (scientific) => {
```

Comment:
Oh man, names like `scientific` makes me really want Typescript in our codebase. I have no idea what this parameter is supposed to represent upon first inspection. Had to search around to find that it's referring to scientific name. maybe `scientificName` or alternative is more informative?

Reply:
This method might need a comment describing why we would need to fix scientific for image